common/v2/features/AddAccount/stories.tsx

```diff
@@ -23,47 +21,38 @@ export const IS_NOT_ELECTRON_AND_IS_NOT_DEV: boolean = !IS_ELECTRON && !IS_DEV;
 
 export const getStories = (): IStory[] => [
   {
-    name: WalletId.WEB3,
```

Comment:
Why is the ordering changed here?

Reply:
In order to have 2-3-3 I had to set `flex-wrap` to `wrap-reverse` so I needed the list to be reversed

Reply:
It should be 2-2-3-1, like I said in my other comment 😄